README.md

```diff
@@ -31,7 +31,30 @@ TODO: INSTALLATION AND SETUP
 
 ## Application Structure
 
-TODO: DOCUMENT APPLICATION STRUCTURE
+The application structure below is a starting point for the root project structure. This will be built upon as
+standards are implemented for deeper directories.
+
+*DEV: When proposing standards that affect the application structure, please update this tree.*
+
+
+```
+.
+├── bin                      # Build/Start scripts
+├── dist                     # The folder for the compiled output (ignored by git)
+├── docs                     # Documentation files for the project
```

Comment:
That's a good question - I would propose marking it as optional. Here's my thinking:

I've seen README files used differently - ones that have all of the documentation in the single README file w/ a ToC, and others that have the very basic "getting started" info and then a link to a `docs/index.md` file which serves as an entry point to the docs folder containing the documentation separated into different files.

And some have a ton of content in the main README AND also have a docs folder with documentation on more complicated features or documentation that is used less often.

I think most of our projects for clients likely won't end up using a docs folder... but in the event that they would, we at least have the location and dirname set.

As far as what it's doing that the other READMEs aren't? For this starter kit specifically, I think it would be a place to put the documentation that doesn't belong in a directory-specific README file... or a place to document the things that we don't want to have clutter up the other docs, but that we want to document in the event that someone else runs into a similar problem in the future.

OR, if we decided that we didn't want all of those sections in the main README in the one file, we could split the documentation into that folder.

Thoughts? Since we're not using it now, and we'll be marking it optional, perhaps I update the description `Documentation files for the project` to be something more along the lines of `Supplemental documentation files for project if needed`